Autocomplete/predefined-items-and-recent-searches/src/style.css

```diff
@@ -0,0 +1,168 @@
+* {
```

Comment:
Do we want to provide a default css file for autocomplete-js? cc @francoischalifour

Reply:
Yes—we starting working on a theme with Kevin before the whole Documentation Search v2. Hopefully we'll starting working back on it soon.